diff --git a/chrome/install_static/install_modes.cc b/chrome/install_static/install_modes.cc
index 8454b2ab6871c950ed976aafdbec8cfacb506e95..0a711b39e9b713e4421140fdd192a5d757dee98c 100644
--- a/chrome/install_static/install_modes.cc
+++ b/chrome/install_static/install_modes.cc
@@ -24,6 +24,20 @@ std::wstring GetClientStateMediumKeyPathForApp(const wchar_t* app_guid) {
   return std::wstring(L"Software\\Google\\Update\\ClientStateMedium\\")
       .append(app_guid);
 }
+#elif defined(BRAVE_CHROMIUM_BUILD)
+std::wstring GetClientsKeyPathForApp(const wchar_t* app_guid) {
+  return std::wstring(L"Software\\BraveSoftware\\Update\\Clients\\").append(app_guid);
+}
+
+std::wstring GetClientStateKeyPathForApp(const wchar_t* app_guid) {
+  return std::wstring(L"Software\\BraveSoftware\\Update\\ClientState\\")
+      .append(app_guid);
+}
+
+std::wstring GetClientStateMediumKeyPathForApp(const wchar_t* app_guid) {
+  return std::wstring(L"Software\\BraveSoftware\\Update\\ClientStateMedium\\")
+      .append(app_guid);
+}
 #else
 std::wstring GetUnregisteredKeyPathForProduct(const wchar_t* product) {
   return std::wstring(L"Software\\").append(product);
