diff --git a/chrome/browser/permissions/permission_manager_factory.cc b/chrome/browser/permissions/permission_manager_factory.cc
index 7972a184d4ce12eeae4c95f487ac04992f74e5ab..b24b296e7f49a3fc4f0289655150dfe23e963aad 100644
--- a/chrome/browser/permissions/permission_manager_factory.cc
+++ b/chrome/browser/permissions/permission_manager_factory.cc
@@ -159,7 +159,7 @@ KeyedService* PermissionManagerFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
   Profile* profile = Profile::FromBrowserContext(context);
   return new permissions::PermissionManager(profile,
-                                            CreatePermissionContexts(profile));
+    BRAVE_PERMISSION_MANAGER_FACTORY_BUILD_SERVICE_INSTANCE_FOR);
 }
 
 content::BrowserContext*
