diff --git a/third_party/blink/renderer/core/svg/svg_path_element.cc b/third_party/blink/renderer/core/svg/svg_path_element.cc
index c57638d1bb9300cfdf61f00f0113661fd4ff4ac3..be4aabcfcf27fe5d0ac1d862522d9ab3f542e694 100644
--- a/third_party/blink/renderer/core/svg/svg_path_element.cc
+++ b/third_party/blink/renderer/core/svg/svg_path_element.cc
@@ -65,6 +65,7 @@ Path SVGPathElement::AsPath() const {
 }
 
 float SVGPathElement::getTotalLength(ExceptionState& exception_state) {
+  BRAVE_SVG_PATH_ELEMENT_GET_TOTAL_LENGTH
   GetDocument().UpdateStyleAndLayoutForNode(this,
                                             DocumentUpdateReason::kJavaScript);
   return SVGPathQuery(PathByteStream()).GetTotalLength();
