diff --git a/third_party/blink/renderer/core/svg/svg_text_content_element.cc b/third_party/blink/renderer/core/svg/svg_text_content_element.cc
index 542db849b2a919947b339fd50164c57128a5dde3..672ae45bed3f70e9c9422997ae809d7a8bcc4c8b 100644
--- a/third_party/blink/renderer/core/svg/svg_text_content_element.cc
+++ b/third_party/blink/renderer/core/svg/svg_text_content_element.cc
@@ -98,6 +98,7 @@ unsigned SVGTextContentElement::getNumberOfChars() {
 }
 
 float SVGTextContentElement::getComputedTextLength() {
+  BRAVE_SVG_TEXT_CONTENT_ELEMENT_GET_COMPUTED_TEXT_LENGTH
   GetDocument().UpdateStyleAndLayoutForNode(this,
                                             DocumentUpdateReason::kJavaScript);
   return SVGTextQuery(GetLayoutObject()).TextLength();
